core/src/main/java/io/grpc/internal/MutableHandlerRegistry.java

```diff
@@ -29,7 +29,9 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.grpc;
+package io.grpc.internal;
```

Comment:
You can pass one to `NettyServerBuilder.forRegistryAndPort()`. I'm not necessarily against making it internal in the short-term, but we hadn't discussed that at all.